addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/command/TeslaCommandExtension.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ * <p>
```

Comment:
Did you generated those headers through `mvn license:format`? Afaik, we do not have that `<p>` in there. Could you remove it here (and in the other modified headers)?